packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/NftForm.types.ts

```diff
@@ -20,8 +21,8 @@ export type NftFormFields = {
   royalty?: number
   startingPrice?: number
   buyNowPrice?: number
-  whitelistedMembersIds?: string[] // to be used later
   auctionDurationBlocks?: number
+  whitelistedMembers?: BasicMembershipFieldsFragment[] // to be used later
```

Comment:
@drillprop I think we can remove the comment now?
```suggestion
  whitelistedMembers?: BasicMembershipFieldsFragment[]
```